chrome/content/zotero/itemTree.jsx

```diff
@@ -3821,12 +3821,12 @@ var ItemTree = class ItemTree extends LibraryTree {
 	_getTagSwatch(tag, color) {
 		let span = document.createElement('span');
 		span.className = 'tag-swatch';
-		// If only emoji, display directly
+		// If contains emojis, display directly
 		//
 		// TODO: Check for a maximum number of graphemes, which is hard to do
 		// https://stackoverflow.com/a/54369605
-		if (Zotero.Utilities.Internal.isOnlyEmoji(tag)) {
-			span.textContent = tag;
+		if (Zotero.Utilities.Internal.containsEmoji(tag)) {
+			span.textContent = Zotero.Tags.extractEmojiForItemsList(tag);
```

Comment:
I guess we don't really need `containsEmoji` at all, right? It's more or less doing the same regex, so why do it twice instead of once?